src/pages/execution/tabLoader.tsx

```diff
@@ -255,6 +261,73 @@ export default function tabLoader(tabBase: TabBase): TabData {
   };
 }
 
+const generateAlerts = (tasks) => {
+  const alerts = [];
+  tasks.forEach((taskResult) => {
+    //detect multiple polls and callbackafterseconds >0
+    if (taskResult.pollCount > 1) {
+      const formatedDate =
+        !isNaN(taskResult.updateTime) && taskResult.updateTime > 0
+          ? timestampRenderer(taskResult.updateTime)
+          : "N/A";
+      let alertMessage = `Task "${taskResult.taskDefName}" has been polled by workers for ${taskResult.pollCount} times.`;
+      if (taskResult.callbackAfterSeconds > 0)
+        alertMessage += ` A callback is set to ${taskResult.callbackAfterSeconds} seconds at ${formatedDate}`;
+      alerts.push(<Alert severity="warning">{alertMessage}</Alert>);
+    }
+  });
+
+  return alerts;
+};
+
+function AlertComponent() {
+  const tileFactoryContext = useContext(TileFactoryContext).executionAndTasks;
```

Comment:
might want to rename this so its not confused to be the actual context object - for readability